src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -16,6 +16,7 @@ export default Vue.extend({
   data: function () {
```

Comment:
Would you mind going up and changing
`name: 'FtElementList'`
to
`name: 'PlaylistInfo'`
while you're at it? Thanks :^)